openeo/internal/jupyter.py

```diff
@@ -1,16 +1,89 @@
 import json
 
-SCRIPT_URL = 'https://cdn.jsdelivr.net/npm/@openeo/vue-components@2.0.0-rc.2/assets/openeo.min.js'
+SCRIPT_URL = 'https://cdn.jsdelivr.net/npm/@openeo/vue-components@2/assets/openeo.min.js'
 COMPONENT_MAP = {
+    'collection': 'data',
+    'data-table': 'data',
     'file-format': 'format',
     'file-formats': 'formats',
+    'item': 'data',
+    'items': 'items',
```

Comment:
"item"/"items" sounds very generic, can't we make it a bit more concrete and self-describing, like "stac item" or "collection item"? Or is it really generic/abstract at this point?

Reply:
This is a mapping of component and parameter for the Vue component. This directly translates into code, we can't change anything here.

Reply:
> No user will see it.

I'm not sure, if they want to work with `collection_items` results outside of a jupyter context, they;'ll will have to understand what  `VisualDict("items"` is about

Reply:
Maybe, but as I said, that's the API of the vue-components and there's nothing we can change about it (except adding a comment behind the line). On the other hand, VisualDict is just a dict?!

Reply:
I just realized I can remove the line completely.